lib/evm_watchdog.rb

```diff
@@ -1,95 +0,0 @@
-require 'time'
-require 'linux_admin'
-require 'fileutils'
-
-module EvmWatchdog
-  PID_LOCATION = '/var/www/miq/vmdb/tmp/pids/evm.pid'.freeze
-  UPDATE_FILE  = '/var/www/miq/vmdb/tmp/miq_update'.freeze
-
-  def self.check_evm
-    pid_file = read_pid_file(PID_LOCATION)
-    pid_ps = get_ps_pids('MIQ Server')
-    if pid_file.nil?
-      # EVM exited gracefully - no pid file, nothing to do.
-      # Future TODO - Check database to see if EVM should be started?
-    elsif pid_file.empty? # Not really sure how we got an empty pid file, but we'll just log it for now.
-      log_info("Detected an empty PID file for EVM Server Process.")
-    elsif pid_file.include? "no_db"
-      db_state = get_db_state # TODO: - This is here because it is costly.  Should be moved up when not as costly.
-      if db_state.empty?
-        # Database down, nothing to do but wait for it to come up.
-      else
-        log_info("Detected that the database is now available.")
-        start_evm
-      end
-    elsif pid_file.to_i == 0
-      log_info("Detected non-numeric PID file contents: #{pid_file}")
-    elsif pid_ps.include?(pid_file.to_i)
-      # If the list of pids in ps includes the pid in the file, EVM is running normally
-    else
-      db_state = get_db_state # TODO: - See note above.
-      if db_state.empty?
-        log_info("Detected that the database is down.")
-      elsif ["started", "starting"].include?(db_state)
-        log_info("Detected that the EVM Server with PID [#{pid_file.to_i}] is no longer running.")
-        start_evm
-      else # Not sure why we have a pid file here, maybe we should remove it?
-        log_info("Detected a PID file: [#{pid_file}], but server state should be: [#{db_state}]...")
-      end
-    end
-  end
-
-  def self.check_for_update
-    return unless File.exist?(UPDATE_FILE)
-    packages = File.read(UPDATE_FILE).split
-    LinuxAdmin::Yum.update(*packages)
```

Comment:
This is used by [update_management.rb](https://github.com/ManageIQ/manageiq/blob/61c77793f44a2453c0908997d4a8c5f79607faa3/app/models/miq_server/update_management.rb) to update our code.

for some reason I thought we got out of the business of doing this.
I don't see where they are called